### PR TITLE
Migrating the plan upgrade troubleshooting article

### LINKED ISSUE #800: Migrate article: Having trouble creating or upgrading to a larger plan?
Migrate the content from this article to fit the documentation portal format https://help.aiven.io/en/articles/5240633

(Assign this issue to yourself before you start, and remember to mention the issue number when you open your pull request)

### PR DESCRIPTION
**Pull request by sluhur**
# What changed, and why it matters

Fixes #800 
Original link: https://help.aiven.io/en/articles/5240633

### CODE REVIEW

**Comment by sluhur** [comment]
This is for migration #800 
https://github.com/aiven/devportal/issues/800

**Comment by lornajane** [comment]
I made some minor edits to the title and description of the pull request to make it easier to track and understand in our big list of active pull requests.

File: docs/platform/concepts/plan-upgrades.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Prepare for larger service plans
```
Comment (ftisiot) [inline docs/platform/concepts/plan-upgrades.rst:1]: I believe the title could be more informative? The article seems to be related to billing and credit card details but the title seems to point people to an howto move to a larger plan?

**Review by ftisiot — CHANGES_REQUESTED** [review]
Hi! Thanks for the contribution!

The article is readable however I would like to move to a more informative title, since the current one could be misleading (makes me think about how to do upgrades).

Is this article about billing? Maybe it should go under the billing section?

Also, I would add links to sections where we explain how to perform the upgrade

File: docs/platform/concepts/plan-upgrades.rst (hunk 1 above)
Comment (sluhur) [reply docs/platform/concepts/plan-upgrades.rst:1]: Maybe the title should be "Solving issues when upgrading to larger service plans"

**Comment by ftisiot** [comment]
Moved the article to the howto/billing section, changed the style of the console warning message (a ..warning could be seen as a worning from the devportal) and rebased. merging it now

**Review by ftisiot — APPROVED** [review]
LGTM

**Pull request merged**